.gitignore

```diff
@@ -1,4 +1,4 @@
 .idea/
 node_modules/
 .publish/
-npm-debug.log
+npm-debug.log*
```

Comment:
I had a file on my system called `npm-debug.log.1182010857`, so I updated the .gitignore pattern. :)